### ISSUE TITLE
Is Data Streams Monitoring supported with this Lambda extension?

### ISSUE THREAD

**Issue by alank-ps** [DataDog/datadog-lambda-extension#489]
Hi :wave: I have been trying to use DSM https://docs.datadoghq.com/data_streams/go/ in a golang lambda application to produce stuff to Kafka. However, the DSM is not enabled in the service page in my Datadog account after deploying it. I have my env variable `DD_DATA_STREAMS_ENABLED=true`. Traces and logs are successfully shipped so definitely it is connecting to Datadog.

Is it possible Data Streams Monitoring not supported yet?

**Comment by piochelepiotr** [comment]
Hi @alank-ps , are you using one of the two libraries we support?
https://github.com/confluentinc/confluent-kafka-go
or
Sarama (that now moved to https://github.com/IBM/sarama), and wrapping the producer in the way described here:
https://docs.datadoghq.com/data_streams/go/#confluent-kafka-client ?

On the consuming side, for lambda, AWS will directly call your lambda, so you wouldn't use the consumer. This is not something Data Streams supports yet. You would need to use our manual instrumentation in this case: https://docs.datadoghq.com/data_streams/go/#manual-instrumentation

The producer side should work (if you use a recent lambda layer). If it doesn't, can you please open a support ticket so we can ask for a bit more details and investigate?
Thanks!

**Comment by alank-ps** [comment]
Yes I am using the confluent lib and I am having consumer on ECS instead of lambda, which works just fine. But the producer deployed as a lambda, doest have the DSM enabled on the service page. I couldn't tell if it's a config issue or not (the first few logs about the datadog agent config doesn't tell anything about DSM). 

Thanks I will gather more detailed info and will create a support ticket.
